lib/beaker/host_prebuilt_steps.rb

```diff
@@ -329,7 +329,7 @@ def copy_ssh_to_root host, opts
           host.exec(Command.new('cp -r .ssh /cygdrive/c/Users/Administrator/.'))
           host.exec(Command.new('chown -R Administrator /cygdrive/c/Users/Administrator/.ssh'))
         elsif host['platform'] =~ /windows/ and not host.is_cygwin?
-          host.exec(Command.new("if exist .ssh (xcopy .ssh C:\\Users\\Administrator\\.ssh /s /e)"))
+          host.exec(Command.new("if exist .ssh (xcopy .ssh C:\\Users\\Administrator\\.ssh /s /e /y /i)"))
```

Comment:
Test was added.

But I didn't see any examples in other tests how properly add doc references.